src/stats/probs/adaptive_integration.rs

```diff
@@ -43,7 +43,7 @@ use ordered_float::NotNan;
 /// );
 /// abs_diff_eq!(integral.exp(), 0.682, epsilon=0.01);
 /// ```
-pub fn ln_integrate_exp<T, F>(
+pub fn ln_integrate_exp<T, F, E>(
```

Comment:
I think a PR changing the signature of a public method like this (which, granted, in most cases won't actually affect most users because the type will just be inferred, but will affect anyone who needs to specify any of the generic parameters) should probably not be labeled "chore", since "chore" PRs don't show up in the changelog.

Reply:
Would you classify it at a `build:` then?

Reply:
I think it should be feat: actually, since the API is extended with a new feature here (allowing to have a fallible conversion from f64)